src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -325,7 +325,9 @@ function warnAboutUnstableUse() {
   warned = true;
 }
 
+/* eslint-disable max-len */
 function renderSubtreeIntoContainer(parentComponent : ?ReactComponent<any, any, any>, children : ReactNodeList, containerNode : DOMContainerElement | Document, callback: ?Function) {
+/* eslint-enable max-len */
```

Comment:
Rather than bypassing the lint check, seems better to split params across lines:

```js
function renderSubtreeIntoContainer(
  parentComponent : ?ReactComponent<any, any, any>,
  children : ReactNodeList, containerNode : DOMContainerElement | Document,
  callback: ?Function
) {
```

This is what we normally do (eg [`createInstance `](https://github.com/facebook/react/blob/3c75593692524678fbb74e8077018cc39b04ffaa/src/renderers/dom/fiber/ReactDOMFiber.js#L171-L177) above).